docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -1,258 +1,224 @@
 ---
 title: "Go (pull mode)"
 menuTitle: "Go (pull mode)"
-description:  "Instrumenting Golang applications for continuous profiling"
+description: "Set up Go profiling in pull mode"
```

Comment:
Quotes aren't necessary in the description, title, and menuTitle unless there are special characters (like parentheses). 
```suggestion
description: Learn how to set up Go profiling in pull mode.
```